apps/client/src/features/operator/operator.options.tsx

```diff
@@ -6,12 +6,10 @@ import { ViewOption } from '../../common/components/view-params-editor/types';
 export const getOperatorOptions = (customFields: CustomFields, timeFormat: string): ViewOption[] => {
   const fieldOptions = makeOptionsFromCustomFields(customFields, { title: 'Title', note: 'Note' });
 
-  const customFieldSelect = Object.entries(customFields).reduce<
-    Record<string, { value: string; label: string; colour: string }>
-  >((acc, [key, field]) => {
-    acc[key] = { value: key, label: field.label, colour: field.colour };
-    return acc;
-  }, {});
+  const customFieldSelect = Object.entries(customFields).map(([key, value]) => ({
+    value: key,
+    label: value.label,
+  }));
```

Comment:
not related to this code, but the highlight field select in the op parameters does not work for me

Reply:
Hopefully, resolved in https://github.com/cpvalente/ontime/pull/1438/commits/c496c3658d431fed64b92f1f647895535b21c7e9